src/components/Link/__stories__/Link.docs.mdx

```diff
@@ -0,0 +1,63 @@
+import {Props} from '@storybook/addon-docs';
```

Comment:
It is not used. It might be better to use `ArgsTable` (`Props` is an alias for it) because it is mentioned in the storybook docs.

Reply:
done